.github/workflows/build.yml

```diff
@@ -12,20 +12,6 @@ on:
       - '**.md'
 
 jobs:
-  build:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: false
-      matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
-    steps:
-      - uses: actions/checkout@v3
-      - name: Setup Node.JS 16
-        uses: actions/setup-node@v3
-        with:
-          node-version: 16.x
-          cache: npm
-      - run: npm ci
-      - run: npm run build
-      - run: npm run format-check
-      - run: npm test
\ No newline at end of file
+  call-basic-validation: # The reusable workflow can be found here: https://github.com/actions/reusable-workflows/blob/main/.github/workflows/basic-validation.yml
+    name: Basic validation
+    uses: actions/reusable-workflows/.github/workflows/basic-validation.yml@main
```

Comment:
I think we don't need such comments because it's obvious where the reusable workflow file is, judging by the value of the `uses` key